lib/trento/discovery/payloads/cluster/cluster_discovery_payload.ex

```diff
@@ -177,25 +213,16 @@ defmodule Trento.Discovery.Payloads.Cluster.ClusterDiscoveryPayload do
   end
 
   defp parse_cluster_additional_sids(%{
-         "cluster_type" => ClusterType.hana_scale_up(),
-         "cib" => %{"configuration" => %{"resources" => %{"primitives" => primitives}}}
-       }),
-       do: get_sapinstance_sids(primitives)
-
-  defp parse_cluster_additional_sids(%{
-         "cib" => %{"configuration" => %{"resources" => %{"clones" => nil, "groups" => groups}}}
+         "cib" => %{
```

Comment:
I have simplified this function to get additional sids from resources and grouped resources.